errors/missing-env-value.md

```diff
@@ -0,0 +1,21 @@
+# Missing Env Value
+
+#### Why This Error Occurred
+
+On one of your pages' config you requested an env value that wasn't populated.
+
+```js
+export const config = {
+  env: ['MY_ENV_KEY'],
```

Comment:
Not sure this example adds value. We should add a comment explaining that it's missing from `.env`, and show the corresponding `.env` file example.

Reply:
Updated